valhalla/mjolnir/util.h

```diff
@@ -3,35 +3,41 @@
 
 #include <boost/property_tree/ptree.hpp>
 #include <list>
+#include <map>
 #include <string>
-#include <unordered_map>
```

Comment:
we still use this header as the class is used in this file, though we fail at it we should keep trying to adhere to the include what you use idiom

Reply:
Fixed